src/package/i18n/en.json

```diff
@@ -1,245 +1,245 @@
 {
-    "AddButton.plus": "+",
-    "Banner.EditImageDialog.Title": "Pick an image",
-    "Banner.EditImageDialog.unsplashButton": "Find on unsplash",
-    "Banner.actions.customize": "Customize",
-    "Banner.actions.customize.dialog.title": "Customize your profile",
-    "Banner.image.editButton": "Edit banner image",
-    "Basics.Back.CodingYears.noExperience": "I've just started coding",
-    "Basics.Back.CodingYears.title": "Code Experience",
-    "Basics.Back.CodingYears.value": "{codingYears, plural, one {# year} other {# years}} coding (every experiences, studies, personal projects, work...)",
```

Comment:
Regarding the en.json changes: I used a GUI translator to translate the strings faster and while it was saving, it did some syntax improvements to the json file which resulted in this space changes.